unblob/handlers/compression/zstd.py

```diff
@@ -0,0 +1,43 @@
+import io
+import typing
+from typing import IO, Optional
+
+import zstandard
+from structlog import get_logger
+
+from unblob.extractors import Command
+
+from ...file_utils import InvalidInputFormat
+from ...models import File, Handler, HexString, ValidChunk
+
+logger = get_logger()
+MIN_READ_RATIO = 0.1
+
+
+class ZSTDHandler(Handler):
+    NAME = "zstd"
+
+    PATTERNS = [HexString("28 B5 2F FD")]
+
+    EXTRACTOR = Command("zstd", "-d", "{inpath}", "-o", "{outdir}/{infile}")
+
+    def calculate_chunk(self, file: File, start_offset: int) -> Optional[ValidChunk]:
+
+        file.seek(start_offset, io.SEEK_SET)
+        decompressor = zstandard.ZstdDecompressor()
+        chunks = 0
```

Comment:
we would be able to verify the frame header to make sure we do not pick up false-positive matches based on the magic.